admin/app/clusters/[id]/DataTable.tsx

```diff
@@ -22,6 +22,10 @@ export const DataTable = <T = any,>({
   noDataMessage?: string;
   columnDef?: ColumnDef<T>[];
 }) => {
+  if (data === undefined || data.length === 0) {
+    return <div className="text-center">{noDataMessage}</div>;
```

Comment:
Thank you. Can we update so that it follows the style guide?

```suggestion
    return <p className="text-gray-400 mt-2">{noDataMessage}</p>;
```

Reply:
Also, this needs to be rendered after the `useReactTable()` hook 🙏🏽